drake/examples/Pendulum/CMakeLists.txt

```diff
@@ -8,27 +9,33 @@ include_directories(${PROJECT_SOURCE_DIR}/util)
 if (LCM_FOUND AND NOT WIN32)  # win32 won't be hard... just have to add declspecs
   include_directories(${PROJECT_SOURCE_DIR}/pod-build/lcmgen)
   add_executable(runPendulumDynamics runDynamics.cpp)
-  target_link_libraries(runPendulumDynamics drakeSystem drakeRBM)
+  target_link_libraries(runPendulumDynamics drakeRBM)
+  add_dependencies(runPendulumDynamics drake_lcmtypes lcmtype_agg_hpp)
+  pods_use_pkg_config_packages(runPendulumDynamics lcm)
```

Comment:
We should have a macro for the above two lines. I know haven't been doing this right myself.